Src/ILGPU/Runtime/ExchangeBuffer.cs

```diff
@@ -277,15 +295,41 @@ public void CopyFromAccelerator(AcceleratorStream stream)
         /// <param name="extent"></param>
         /// <returns>The view.</returns>
         public ArrayView2D<T> As2DView(Index2 extent) =>
-            CPUView.BaseView.As2DView<T>(extent);
+            cpuView.BaseView.As2DView<T>(extent);
 
         /// <summary>
         /// Gets the part of this buffer on CPU memory as a 2D View.
         /// </summary>
         /// <param name="extent"></param>
         /// <returns>The view.</returns>
         public ArrayView3D<T> As3DView(Index3 extent) =>
-            CPUView.BaseView.As3DView<T>(extent);
+            cpuView.BaseView.As3DView<T>(extent);
+
+        /// <summary>
+        /// Gets this exchnage buffer as a <see cref="Span{T}"/>, copying from the
+        /// accelerator in the process
```

Comment:
MINOR: Spelling and full stop.

```suggestion
        /// Gets this exchange buffer as a <see cref="Span{T}"/>, copying from the
        /// accelerator in the process.
```

Reply:
I misspell that word around 80 times on a daily basis for some reason